scripts/test

```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+cd $(dirname $0)/..
+
+helm init -c
+helm plugin install https://github.com/lrills/helm-unittest
```

Comment:
I think Line 7 and 8 are really install process and should moved into the Dockerfile.  

Just to reduce on my silly errors I'm linking the test script as a pre-commit hook in git. How important is it to split out the `helm lint` into its own step?